libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/ScreenLockActivity.java

```diff
@@ -128,6 +127,7 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+    @SuppressLint("MissingSuperCall")
```

Comment:
This was an inspector error, though everyone was compiling fine.  I thought it worth a quick out of context fix up!